lib/engine/game/g_1867/game.rb

```diff
@@ -1174,7 +1174,7 @@ def sorted_corporations
         end
 
         def unstarted_corporation_summary
-          unipoed = @corporations.reject(&:ipoed)
+          unipoed = (@corporations + @future_corporations).reject(&:ipoed)
```

Comment:
why is this copy and pasted from 1861, isn't 67 a subclass of 61? do we even need this method?

same with ireland, the ireland code is better because it uses partition, can you clean that up?

Reply:
This is the first time I've looked at any of three three titles and I don't know their rules; I was just trying to clean up an easy issue. The 1861 and 1867 `unstarted_corporation_summary` methods are slightly different in the way they present the information (the string they return); someone who knows the titles might be able to say why. It doesn't seem that 1861 or 1867 can simply use `partition` because they have a third type called `:national`.

I could make a 1867 helper method that does almost all of the work and then separate methods for 1861 and 1867 that format the results differently as a string if you would prefer that. I think Ireland has to stay separate.